eclass/git-r3.eclass

```diff
@@ -120,6 +120,12 @@ fi
 # usage.
 : "${EGIT_LFS_CLONE_TYPE:=shallow}"
 
+# @ECLASS_VARIABLE: GDIR_NAME
```

Comment:
1. Prefix it properly with `EGIT_`.
2. Mark as `@OUTPUT_VARIABLE`.
3. Make it an array.